### ISSUE TITLE
v2.0b3 error on the 'flatten' function

### ISSUE THREAD

**Issue by LucaNap** [KeplerGO/lightkurve#916]
I am using Windows and the latest version of Lightkurve (dev), and I ran into the following issue while trying to call the flatten function: 

```python
Traceback (most recent call last):
  File "D:\Anaconda3\lib\site-packages\astropy\time\core.py", line 430, in _get_time_fmt
    return cls(val, val2, scale, precision, in_subfmt, out_subfmt)
  File "D:\Anaconda3\lib\site-packages\astropy\time\formats.py", line 149, in __init__
    val1, val2 = self._check_val_type(val1, val2)
  File "D:\Anaconda3\lib\site-packages\astropy\time\formats.py", line 430, in _check_val_type
    raise TypeError(
TypeError: for jd class, input should be doubles, string, or Decimal, and second values are only allowed for doubles.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\Python38\test GAPS\light2.py", line 62, in <module>
    flat, trend = folded_lc.flatten(window_length=301, return_trend=True)
  File "D:\Anaconda3\lib\site-packages\lightkurve\lightcurve.py", line 663, in flatten
    flatten_lc = self.copy()
  File "D:\Anaconda3\lib\site-packages\astropy\table\table.py", line 3108, in copy
    out = self.__class__(self, copy=copy_data)
  File "D:\Anaconda3\lib\site-packages\lightkurve\lightcurve.py", line 203, in __init__
    data['time'] = Time(data['time'],
  File "D:\Anaconda3\lib\site-packages\astropy\time\core.py", line 1526, in __init__
    self._init_from_vals(val, val2, format, scale, copy,
  File "D:\Anaconda3\lib\site-packages\astropy\time\core.py", line 372, in _init_from_vals
    self._time = self._get_time_fmt(val, val2, format, scale,
  File "D:\Anaconda3\lib\site-packages\astropy\time\core.py", line 438, in _get_time_fmt
    raise ValueError(
ValueError: Input values did not match the format class jd:

TypeError: for jd class, input should be doubles, string, or Decimal, and second values are only allowed for doubles.
```

I was able to bypass the error by commenting the following lines in 'lightkurve.py':

```python
if data and 'time' in data.keys():
  if not isinstance(data['time'], Time):
     data['time'] = Time(data['time'],
     format=deprecated_kws.get("time_format", self._default_time_format),
     scale=deprecated_kws.get("time_scale", self._default_time_scale))
```

**Comment by barentsen** [comment]
Oops!  Thank you for reporting this @LucaNap! :+1:

I am not able to reproduce this so far. For example, the following snippet works fine for me:

```python
import lightkurve as lk
lc = lk.search_lightcurve("Proxima Cen", sector=11).download()
lc.flatten()
```

Would you be willing and able to post an as-minimal-as-possible snippet of code that triggers the error on your machine?

Thanks++!

**Comment by LucaNap** [comment]
For example:

target = 'TOI 182.01'
lc = lk.search_lightcurve(target,mission='TESS').download_all(download_dir='data/'+target).stitch()
folded_lc = lc.remove_outliers().fold(period=X, epoch_time=Y)
flat, trend = folded_lc.flatten(window_length=301, return_trend=True)

**Comment by barentsen** [comment]
Oh dear, this snippet works fine for me using the current master branch of Lightkurve.

The error you encounter appears to originate in `astropy\time\core.py`, so it would be useful to know which version of astropy you are using.  You can find out as follows:

```python
>>> import astropy
>>> print(astropy.__version__)
4.1
```

Can you let me know what version number you see? Thanks++!

**Comment by LucaNap** [comment]
Apparently I am using 4.2rc1, which was downloaded by anaconda automatically when I installed Lightkurve!

**Comment by barentsen** [comment]
Ah!  I am able to reproduce the exact same error when I install astropy 4.2rc1 in a virtual environment.

Thanks!  That's progress! :+1:

**Comment by barentsen** [comment]
Thanks so much for reporting this bug @LucaNap!  🙏 

The issue was caused by a minor change in AstroPy v4.2.

I implemented a fix in #917 which has now been merged into the master branch.  I expect to release `2.0b4` soon to include this fix alongside a number of other recent changes.
